zip-0316.rst

```diff
@@ -21,9 +21,9 @@
 Terminology
 ===========
 
-The key words "MUST", "MUST NOT", "SHOULD", and "MAY" in this document are
-to be interpreted as described in BCP 14 [#BCP14]_ when, and only when, they
-appear in all capitals.
+The key words "MUST", "MUST NOT", "SHOULD", "MAY", and "RECOMMENDED" in this
```

Comment:
```suggestion
The key words "MUST", "MUST NOT", "SHOULD", "RECOMMENDED", and "MAY" in this
```
(Because SHOULD and RECOMMENDED mean the same thing.)